pgjdbc/src/main/java/org/postgresql/jdbc/PgStatement.java

```diff
@@ -528,7 +528,7 @@ public void setMaxRows(int max) throws SQLException {
     checkClosed();
     if (max < 0) {
       throw new PSQLException(
-          GT.tr("Maximum number of rows must be a value grater than or equal to 0."),
+          GT.tr("Maximum number of rows must be a value greater than or equal to 0."),
```

Comment:
Message update requires translation file updates

Reply:
Is this just a gradle task ?